packages/rexray/buildinfo.json

```diff
@@ -0,0 +1,9 @@
+{
+  "docker": "golang:1.6",
+  "single_source" : {
+    "kind": "git",
+    "git": "git@github.com:dcos/rexray.git",
+    "ref": "1816ee3868550e07b90f00f57bfd7f737235cd93",
+    "ref_origin": "v0.3.3-dcos"
```

Comment:
if you're not overwriting anything then I'd say publish it now and we can
upload a new version later if needed

On Tue, Jul 19, 2016 at 4:28 PM, lingmann <notifications@github.com> wrote:

> In packages/rexray/buildinfo.json
> <https://github.com/dcos/dcos/pull/347#discussion_r71416809>:
>
> > @@ -0,0 +1,9 @@
> > +{
> > +  "docker": "golang:1.6",
> > +  "single_source" : {
> > +    "kind": "git",
> > +    "git": "git@github.com:dcos/rexray.git",
> > +    "ref": "1816ee3868550e07b90f00f57bfd7f737235cd93",
> > +    "ref_origin": "v0.3.3-dcos"
>
> @jdef <https://github.com/jdef> should we hold off on publishing the
> dvdcli tarball to our downloads bucket until we know more about the
> regression?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/dcos/dcos/pull/347/files/7891f8656e95f5c9cb213e6844e2f096af412136#r71416809>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ACPVLPbyJCxAhpLCYK9T8kurrGkFUfT1ks5qXTNugaJpZM4JNBpJ>
> .
>